.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,63 +1,26 @@
 [Short description explaining the high-level reason for the pull request]
 
-## Additions
+### Additions
 
--
+### Changes
 
-## Removals
-
--
-
-## Changes
-
--
+### Removals
 
 ## Testing
```

Comment:
Let's also add another `#` here as well as on the Checklist header below.

Reply:
Will do!